### ISSUE TITLE
Cannot convert the "Successfully connected to 'Solidfire' with address 'x.x.x.x.x'." value of type "System.String" to type "SolidFire.Core.Objects.SFConnection".

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#24]
v1.4.0.30 on Windows 10

I connect like this:

` $SFConn = Connect-SFCluster -Credential $creds -Target $array -WarningAction SilentlyContinue -ErrorAction stop`

Now ` return $SFConn[0]` and ` return $SFConn` give identical output. And this works:

```
New-SFVolume ... -SFConnection $SFConn[0]
```

But this does not:

```
New-SFVolume ... -SFConnection $SFConn
New-SFVolume : Cannot bind parameter 'SFConnection'. 
Cannot convert the "Successfully connected to 'Solidfire' with address '1.1.1.1'." value of type "System.String" to type "SolidFire.Core.Objects.SFConnection".
At line:1 char:145
+ ... -MaxIOPS 2000 -BurstIOPS 3000 -Enable512e $true -SFConnection $SFConn
+                                                                   ~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-SFVolume], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgumentNoMessage,SolidFire.Volume.New.NewSFVolume
```

Is this a bug?

**Comment by ahaid** [comment]
This is a bug and it will be fixed in the 1.5 version of PowerShell module.

**Comment by ahaid** [comment]
This issue was fixed in the 1.5 release on Nov 29, 2017.
